project.md

```diff
@@ -11,41 +11,59 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+I am helping economists (both professionals and students) to be able to perform financial calculations without having to perform them manually, pay for expensive software, use an Google/Excel sheet for just one line, or look for an X formula calculator and use some weird website. With my DSL implementation, their experience could be improved as they could just type one line of code to perform the calculation as they would have in a Google/Excel sheet.
```

Comment:
I see now how this DSL differs from other uses of simple equations. I like the idea that you take the human error out of writing the equation and plugging in variables.